premake5.lua

```diff
@@ -17,7 +17,7 @@ else
   ARCH = "unknown"
 end
 
-includedirs({
+sysincludedirs({
```

Comment:
`third_party` may count as a system include directory, but why should it be changed for `src` and for the root?